exist/windows.md

```diff
@@ -0,0 +1,128 @@
+- The drive isn't corrupted or have bad sectors
+- Stable power input
+- If you cannot add custom entries in your BIOS boot menu this method may not work for you, read all steps before proceeding
+
+## Situation this applies for
+
+- You have a disk with Windows installed
+- You have enough free space on the disk to support a macOS installation
+
+---
+
+Verify the partition style of the disk is GPT. If you do not know, see: [Checking your disk partition scheme](https://dortania.github.io/OpenCore-Multiboot/exist/data.html#checking-your-disk-partitioning-scheme) and if it is MBR, see: [Converting MBR to GPT](https://dortania.github.io/OpenCore-Multiboot/exist/data.html#converting-mbr-to-gpt).
+
+## Resize existing EFI partition
+
+If your EFI partition is bigger than the default of 100MB (at least 200MB to be safe) which is created by the Windows installation, then you can skip to the next section.
+
+![Windows before partitions](../images/ex-os/before_windows_disk_mgmt.png)
+
+Use a bootable partition manager to resize the EFI partition. To do this you must first move all the partitions in front of it so make sure to backup your data. 
+
+![Gparted original partitions](../images/ex-os/before_gparted.png)
+
+Moving a partition cannot be done while running Windows, so you will need a **bootable** partition manager (free alternative is GParted). This is a potentially destructive operation. 
+
+![Gparted warning](../images/ex-os/move_partition_warning_gparted.png)
+
+Backup your data and proceed at your own risk. If there is not enough free space to move existing partitions to the right, first shrink the existing Windows one. 
+
+![Gparted move Windows right](../images/ex-os/move_windows_right_gparted.png)
+
+Once there is enough free space after the EFI partition, extend it to at least 200MB. 
+
+![Gparted resize EFI](../images/ex-os/resize_efi_gparted.png)
+
+## Resize Windows partition 
+
+If you already have free unallocated space to create the macOS partition, you can skip to the next section. 
+
+Use a partition manager shrink to your Windows partition so that there is enough free space to create a partition that will hold your macOS installation. 
+
+![Gparted shrink Windows](../images/ex-os/shrink_windows_gparted.png)
+
+## Create new exFAT partition
+
+Create a new exFAT **or FAT32** volume on the free unallocated space (GParted does not support exFAT natively until v1.2.0). 
+
+![Gparted create FAT32](../images/ex-os/create_fat32_gparted.png)
+
+If you have other drives you intend to use between both operating systems, now is a good time to format them too. Don't forget to unplug the other drives before proceeding with the macOS installation. 
+
+If you moved partitions around, verify your Windows installation is still functioning before proceeding. If you cannot boot into Windows, restore your backup and try again using a reputable partition manager.
+
```

Comment:
Same comment about using gparted on ntfs. ^

Reply:
Removed mention of Gparted